cfme/tests/control/test_alerts.py

```diff
@@ -32,7 +33,12 @@
 pytestmark = [
     pytest.mark.long_running,
     pytest.mark.meta(server_roles=["+automate", "+smartproxy", "+notifier"]),
-    pytest.mark.uncollectif(lambda provider: provider.one_of(RHEVMProvider)),
+    pytest.mark.uncollectif(
+        lambda provider: (
+            provider.one_of(RHEVMProvider) or
```

Comment:
REQUIRED: off the top of my head, one_of supports iterables like isinstance.
So, I'd ask you to change this statement to something like that
```
provider.one_of((RHEVMProvider, KubeVirtProvider))
```